collector/cpu_linux.go

```diff
@@ -83,8 +114,64 @@ func NewStatCollector() (Collector, error) {
 	}, nil
 }
 
+func (c *cpuCollector) Update(ch chan<- prometheus.Metric) error {
+	err1 := c.UpdateStat(ch)
+	err2 := c.UpdateCpufreq(ch)
+	if err1 != nil {
+		return err1
+	}
+	if err2 != nil {
+		return err2
+	}
+	return nil
+}
+
+// Expose cpu frequency statistics.
+func (c *cpuCollector) UpdateCpufreq(ch chan<- prometheus.Metric) error {
```

Comment:
UpdateCpuFreq -> updateCPUFreq (https://github.com/golang/go/wiki/CodeReviewComments#initialisms and should be unexported)